include/bluetooth/adv_prov/fast_pair.h

```diff
@@ -20,14 +20,14 @@
 extern "C" {
 #endif
 
-/** Set Fast Pair advertising mode.
+/** Set Fast Pair advertising info.
  *
- * User shall make sure that advertising mode is not modified while Fast Pair advertising data
+ * User shall make sure that advertising info is not modified while Fast Pair advertising data
  * provider is providing advertising data.
  *
- * @param[in] fp_adv_mode	Fast Pair advertising mode.
+ * @param[in] fp_adv_info	Fast Pair advertising info.
  */
-void bt_le_adv_prov_fast_pair_mode_set(enum bt_fast_pair_adv_mode fp_adv_mode);
+void bt_le_adv_prov_fast_pair_info_set(struct bt_fast_pair_adv_info fp_adv_info);
```

Comment:
I wonder if we should check for unsupported combinations here and report errors (like discoverable advertising + battery notifications enabled)?

Reply:
Yes, that seems reasonable thing to do since `bt_fast_pair_adv_data_fill` is not called explicitly now.

Reply:
I would stick to just producing warning log in `fp_advertising.c` module and let application continue and ignore battery data in case discoverable advertising is set.